routing/topics.go

```diff
@@ -26,6 +28,9 @@ const (
 
 	remoteCloudTopicFmt     = "devices/%s/messages/devicebound/#"
 	remoteTelemetryTopicFmt = "devices/%s/messages/events/%s"
+
+	localCmdTopicLongFmt  = "command//%s:%s/req//%s"
```

Comment:
I am not sure that I can understand completely the format. What is the representation of the request id?